design/sequence/scheduler-low.puml

```diff
@@ -0,0 +1,108 @@
+@startuml
+
+box "server"
+    participant ":Database" as db
+    participant ":Scheduler" as s
+    participant "j:Job" as j
+    participant ":Dispatcher" as d
+end box
+
+s++
+s -> db++: get_jobs()
+db --> s--: jobs: list[Job]
+
+s -> s++: sorted(jobs)
+note right of s
+    Sorts jobs list by effective priority.
+end note
+s --> s--: jobs
+
+s -> db++: get_available_workers()
+note right of db
+    Return workers with free resources.
```

Comment:
Should return all workers because we might want to schedule an urgent job to a machine without free resources.

Reply:
I also renamed the method to get_work_machines()